changelog.md

```diff
@@ -10,10 +10,11 @@
 
 ## Compiler changes
 
+- Specific warnings can now be turned into errors via `--warningAsError[X]:on|off`.
+- The `define` and `undef` pragmas have been de-deprecated.
 - JavaScript backend adds compile-time optional `let` variable declarations,
```

Comment:
weirdly worded, how about:
JavaScript backend now uses `let` instead of `var` declarations; for previous behavior, compile with `-d:nimJsVar`